engine/src/flutter/lib/web_ui/lib/src/engine/view_embedder/embedding_strategy/full_page_embedding_strategy.dart

```diff
@@ -66,6 +70,10 @@ class FullPageEmbeddingStrategy implements EmbeddingStrategy {
       'meta[name="viewport"]',
     )) {
       assert(() {
+        if (!debugPrintExistingMetaWarning) {
```

Comment:
Again, this is better than nothing, but I'd rather actually fix the root cause of this problem instead of just working around it with a one-off boolean. Is it possible to just fix our test harness so that it either doesn't create this dom node or that it removes it during initialization?

Reply:
There's no root cause. It's just a couple tests that exercise this code path, intentionally creating these duplicate meta tags. The tests are checking the correct outcome and there's no need to see the message printed.

Another way of doing this would be to use the `engine.printWarning(...)` helper instead of `print()`. This is what we do in tests that need to check the output. The test would override `engine.printWarning` and "listen" to what gets printed.

If you think the `engine.printWarning` makes more sense I'm happy to switch to that.

Reply:
Yes, I like that idea, if it's not too much work. Again, I don't consider these blockers but I do like that solution better!